src/styles/_catalog.scss

```
@@ -1,9 +1,23 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 200px);
```

Create a variable for card width, cause you use it in the several places
```suggestion
  grid-template-columns: repeat(auto-fill, $card-width);
```